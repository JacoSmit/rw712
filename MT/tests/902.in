4
3 3
6 8 -10
4 8 6
4 -10 6
3 8
0 2 -10 9 -7 8 2 8
1 9 -3 8 10 8 6 1
-3 3 10 0 6 -10 3 -7
8 3
1 -2 -8
-6 8 -5
-4 1 7
-8 0 8
-5 -4 9
7 7 0
-2 -5 -6
9 -5 7
8 8
10 4 -9 3 3 -4 1 2
6 -10 -7 -9 -8 -7 -7 -4
10 5 -6 5 6 -5 1 10
-9 -9 3 -9 1 -1 -10 0
-2 -9 -2 -7 -9 -4 0 10
-6 9 6 4 1 3 -9 -4
2 -9 6 -1 -7 0 -8 -2
-2 -7 -9 5 0 5 9 -5
