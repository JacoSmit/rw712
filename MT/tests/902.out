1: No
2: Yes
3: Yes
4: Yes
